apps/etl-func/src/adapters/cosmos/event-collector.ts

```diff
@@ -1,63 +1,29 @@
-import {
-  EventCollector,
-  EventsSummary,
-  eventsSummarySchema,
-} from "@/domain/event.js";
+import { EventCollector, EventsSummary } from "@/domain/event.js";
 import { TelemetryEventName, TelemetryService } from "@/domain/telemetry.js";
 import { Container, ItemResponse } from "@azure/cosmos";
+import { v4 } from "uuid";
```

Comment:
use `ulid` instead of `uuid` - it is sortable and it is the default choice for our IDs on IO

Reply:
Ok! Take a look here [bb5417f](https://github.com/pagopa/io-messages/pull/130/commits/bb5417f45db223837c26504402cf7137b554c58a)